### ISSUE TITLE
iOS voice prompts don’t seem to use the system-selected voice

### ISSUE THREAD

**Issue by rachellawson** [osmandapp/OsmAnd#19279]
### Description

I have British Voice 3 (the slightly "generic northern” woman’s voice) selected in iOS 17.4 and that is automatically used in most apps but NOT in OsmAnd for some reason, where a man’s voice is used for navigation prompts.

### Steps to reproduce

Choose a voice in iOS other than the default, navigate and discover the voice doesn’t change in OsmAnd.

### Actual result

Voice is always the same voice, no matter what is chosen in iOS settings

### Expected result

Honour the iOS settings of voice to use.

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version: 4.6.6.1
iOS version: 17.4
Device model: 15 Pro
Crash-Logs: ?
```

**Comment by zipav** [comment]
OsmAnd TestFlight 4.7.2.8

It does not matter what voice is selected in the settings of the iOS device (VoiceOver), in any case the voice prompts are played by a male voice:

https://github.com/osmandapp/OsmAnd/assets/123936521/0c5d264d-2bc3-4656-882d-6fff814f7960

**Comment by rachellawson** [comment]
Yes, that's the issue - should be the same voice used in other apps. As it happens, I'm fairly sure that OutdoorActive had the same problem a long time ago.
